### PR TITLE
bzip2: Update to version 1.0.6.1

### PR DESCRIPTION
**Pull request by brandon93s**

### CODE REVIEW

File: bucket/bzip2.json [hunk 1]
```diff
@@ -21,5 +22,15 @@
     "pre_install": "cp \"$dir\\bzip2.exe\" \"$dir\\bunzip2.exe\"; cp \"$dir\\bzip2.exe\" \"$dir\\bzcat.exe\"",
```
Comment (Ash258) [inline bucket/bzip2.json]: Add binaries for this:

bin: [bzip2.exe, [bzip2.exe, bzcat], [bzip2.exe, bunzip2]]

Comment (brandon93s) [reply bucket/bzip2.json]: Current bin:

```json
    "bin": [
        "bunzip2.exe",
        "bzcat.exe",
        "bzip2.exe",
        "bzip2recover.exe"
    ]
```

What is your proposal?

Comment (Ash258) [reply bucket/bzip2.json]: You do not see problem when you copy same file twice with different name. 

That is nonsense. 

Just use that as i wrote

```yml
bin:
  - bzip2.exe
  - - bzip2.exe
    - bzcat
  - - bzip2.exe
    - bunzip2
```

_Too lazy to quote json_

**Comment by rasa** [comment]
According to https://github.com/philr/bzip2-windows/releases , this release depends on vcredist2015. I wonder is there a way to find that out programmatically, say with cygpath?

**Pull request merged**